LESS/profiles.less

```
@@ -0,0 +1,60 @@
+.team-masthead {
+    .placeholder {
+        height: 20rem;
+        border: 1px dotted black;
+        margin-bottom: 2rem;
+    }
+
+    .banner {
+        font-family: @rway;
+        font-weight: bold;
+        font-size: 1.5rem;
+        line-height: 1.5;
+
+        @media @tablet {
+            font-size: 1.8rem;
+        }
+
+        @media @desktop {
+            font-size: 2rem;
+            padding: 0 1rem;
+        }
+    }
+}
+
+.profile-board {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 2rem;
+
+    @media @tablet {
+        flex-direction: row;
+        flex-wrap: wrap;
+    }
+
+    .profile {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        width: 50%;
+        margin: 1rem 0;
```

```suggestion
        margin: 1em 0; 
```

or switch to percent

---

Honestly, there might not even be a big difference in these cases, depending, however, you probably don't want to use the root font size as the determination of the proportions in cases that involve metrics not involved with fonts....